tests/backward_compatibility.rs

```diff
@@ -0,0 +1,70 @@
+pub mod test_common;
```

Comment:
Don't forget to add copyright.

Reply:
We put a default copyright assignation at the top of `LICENSE`, but I'll add the SPDX-License-Identifier line for consistency.